deepspeed/inference/v2/engine_v2.py

```diff
@@ -104,7 +104,7 @@ def _initialize_tp_group(self):
         ranks = list(range(self._config.tensor_parallel.tp_size))
         return dist.new_group(ranks=ranks)
 
-    def put(self, batch_uids: Iterable[int], batch_tokens: Iterable[torch.Tensor]) -> torch.Tensor:
+    def put(self, batch_uids: Iterable[int], batch_tokens: Iterable[torch.Tensor], do_checks=True) -> torch.Tensor:
```

Comment:
Please add a doc string below indicating what the function of `do_checks`.
```suggestion
    def put(self, batch_uids: Iterable[int], batch_tokens: Iterable[torch.Tensor], do_checks: bool=True) -> torch.Tensor:
```